### ISSUE TITLE
Check building geometry calculations for RC model

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#443]
Check calculations for the different areas for the RC model.
E.g. `A_tot = Lambda_at * A_f, with Lambda_at = 4.5` in 7.2.2.2 in ISO 13790 is different from the current calculation.
This issue is related to #12

**Comment by gabriel-happle** [comment]
Has to be reviewed when higher LOD input is available. Probably modifications will be necessary then.

**Comment by gabriel-happle** [comment]
check relevance with new ISO standard

**Comment by martin-mosteiro** [comment]
Had the following discussion with @gabriel-happle:

In CEA not only do we not account for adiabatic surfaces between buildings, but we even consider them having the same thermal properties as the external surfaces. So if a building is surrounded entirely by another building it will be calculated as a fully exposed windows with heat transfer coefficients equal to `A_wall * U_wall + A_window * U_window + A_basement * U_basement + A_rooof * U_roof` (formula somewhat simplified).
This is a significant oversimplification, and it would it make sense to make the RC model account for the same heated floor area but consider only the exposed surfaces for the transmission loss calculation. One way to do so would be to select which walls are adiabatic by neglecting all walls/windows were the total yearly irradiation is 0, such that `Htr_op` would become `sum(A_wall,i) * U_wall + A_basement * U_basement` for all surfaces `i` such that `sum(I_sol,i) > 0.0` (again, somewhat simplified).
I have implemented this idea on branch [`443-exposed-areas`](https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/443-exposed-areas), however I found that the insolation on adjacent areas was never zero in adjacent buildings in my case study (see issue #2397).
Further development on this is therefore paused until we find a solution to #2397.
